scripts/snapshot_check/run_service.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -e
+
+S3_FOLDER=$BASE_FOLDER/s3
+
+echo "service script"
+
+error=0
+
+check_env () {
```

Comment:
This function definitely deserves an extensive documentation on what it does. :)